### PR TITLE
Let Drush get the base URL for login

### PR DESCRIPTION
**Pull request by amitaibu**
As we might have SSL locally, we can let DDEV and Drush use the best base url

### CODE REVIEW

File: server/scripts/helper-functions.sh [hunk 1]
```diff
@@ -249,8 +241,7 @@ function get_base_url() {
 ##
 function drupal_login {
   cd web
-  BASE_URL="$(get_base_url)"
-  URL=$(ddev exec drush uli --uri "$BASE_URL")
+  URL=$(ddev exec drush uli --uri)
```
Comment (amitaibu) [inline server/scripts/helper-functions.sh]:
```suggestion
  URL=$(ddev exec drush uli)
```

**Review by AronNovak — APPROVED** [review]
Looks good. I'll create a Travis-fixing PR later today into this one, so it will be also green.

**Comment by amitaibu** [comment]
Actually, some parts are missing. Lets discuss :)

**Comment by AronNovak** [comment]
@amitaibu Well, well, not "later today", but this shall be green with the recent merge. Still would like to discuss what's missing or just MWG?

**Comment by amitaibu** [comment]
> Actually, some parts are missing.

The message in the end of the script telling you installation was complete is now missing that URL. However goal is to remove completely the `./install`, so MWG.

**Comment by amitaibu** [comment]
On https://github.com/Gizra/form-wizard-example we have an example of ddev only install

**Comment by AronNovak** [comment]
@amitaibu :+1: , we can port that from there too certainly to get rid of the script.

**Pull request merged**